docs/user_guide/thresholder.rst

```diff
@@ -0,0 +1,560 @@
+.. _thresholder:
+
+***********
+Thresholder
+***********
+
+
+What it does
+~~~~~~~~~~~~
+
+:class:`fairlearn.postprocessing.Thresholder` is a post processing technique
+which outputs a binary prediction based on group-specific thresholds. The
+specified thresholds can work in two directions (positive/negative), and can
+be applied to the output of the :code:`predict()` function a regressor,
+:code:`predict_proba()[:,1]` of a classifier, or on
+:code:`decision_function()` of SVC/LogisticRegression. Examples of these
+different applications can be found below. 
+
+What is a threshold?
+~~~~~~~~~~~~~~~~~~~~
+A threshold is a number which represents the border between positive and
+negative predictions. For most classification algorithms, the default
+threshold is 0.5, which means that a sample with a probability to get 1
+(obtained by calling :code:`estimator.predict_proba()[:,1]`) of 0.6 will be classified
+as 1, and a sample with probability to get 1 of 0.4 will be classified as 0.
+If this threshold would be changed to 0.7, both samples would be classified
+as 0.
+
+
+Why use it
+~~~~~~~~~~~
+:class:`Thresholder` is an easy to interpret tool, that can help explore different
+trade-offs. Where an algorithm like 
+:class:`fairlearn.postprocessing.ThresholdOptimizer` “blindly” satisfies
+a specific constraint, :class:`Thresholder` aids the user in manually exploring
+different trade-offs, which can help gain more insight into the data. Also,
+since :class:`Thresholder` is a post-processing technique, it is not necessary to alter
+an already trained estimator, which might be the only feasible unfairness
+mitigation approach when developers cannot influence training, due to
+practical reasons or security or privacy.
+
+In the next section, we will consider all ways of specifying thresholds, and
+show how to investigate the effect of different thresholds. 
+
+How to use it
+~~~~~~~~~~~~~
+
+The group-specific thresholds are specified in :code:`threshold_dict`. This is
+a dictionary where the keys are the sensitive feature value of the sub-group,
+and the values corresond to the threshold. Below, we will show all different
+kind of ways to specify thresholds.
+
+Furthermore, we will plot some information about the obtained predictions,
+in order to help gain an insight in the way :class:`Thresholder` works.
+
+Dataset
+^^^^^^^^
+The dataset used in all examples below, unless specified otherwise, is a
+clincial dataset of hospital re-admissions over a ten-year period (1998-2008)
+for diabetic patients across 130 different hospitals in the US. Each record
+represents the hospital admission records for a patient diagnosed with
+diabetes whose stay lasted one to fourteen days. We would like to develop a
+classification model, which decides whether the patients should be suggested
+to their primary care physicians for an enrollment into the high-risk care
+management program. The positive prediction will mean recommendation into the
+care program.
```

Comment:
Out of these two options I would prefer the former, mainly because this means that I don’t have to wait for #907, and this can be merged faster. 
To check if I understand correctly, if we would go for this option I should:
-	Move the short description I have here to the dataset user guide
-	Create a separate PR for someone else to complete the description in the dataset user guide
-	Refer to the dataset user guide instead of explaining the dataset in ‘my’ user guide

I should still include the codeblock where I import the dataset and do the train_test_split and stuff in ‘my’ user guide I assume, in order to have the reader understand where all variables later on in the guide are coming from?